src/dao/pages/MultiSigDAODashboard/AssetsList.tsx

```diff
@@ -29,9 +30,11 @@ export function AssetsList() {
   const navigate = useNavigate();
   const handleTransactionSuccess = useHandleTransactionSuccess();
   const depositTokens = useDepositTokens(handleDepositTokensSuccess);
+  const daoSafeIdQueryResults = useContract(safeEVMAddress);
+  const daoSafeId = daoSafeIdQueryResults.data?.data.contract_id ?? "";
```

Comment:
same as previous we could de structure the response either in hook or in mirror node, so it will be in a single place rather than all components.